dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -174,6 +179,31 @@ Create24BppBitmap(HDC hDC, INT cx, INT cy)
     return hbm;
 }
 
+static HBITMAP BitmapFromIcon(HICON hIcon, INT cx, INT cy)
+{
+    HBITMAP hbmRet = NULL;
+    if (HDC hDC = CreateCompatibleDC(NULL))
+    {
+        if (HBITMAP hbm = Create24BppBitmap(hDC, cx, cy))
```

Comment:
I think our coding style does not recommend assignments inside if-s, so I would suggest instead to do the assignment before the if(), then check for NULL.

Reply:
I see. OK.